package/gluon-status-page/javascript/status-page.js

```diff
@@ -133,6 +133,9 @@
 			}
 
 			return 'via ' + addr + ' (unknown iface)';
+		},
+		'tq': function(value) {
+			return formatNumber(100/255 * value, 1) + ' %';
```

Comment:
```suggestion
			return formatNumber(100/255 * value, 1) + '%';
```

removing the space would align it with RAM and fileysystem usage above.